src/repositories/organisations/port.js

```diff
@@ -1,6 +1,7 @@
 /**
  * @typedef {Object} OrganisationsRepository
  * @property {() => Promise<Object[]>} findAll
+ * @property {(orgId: string) => Promise<Object | null>} findByOrgId
```

Comment:
Might be nice to have a contract test that defines the behaviour all adapters implementing the port should exhibit